WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
 		
 		capital = 560 # Cherson
 		
-		hellenic_pagan = 600
-		bosporan_faith = 200
+		hellenic_pagan = 200
+		bosporan_faith = 600
```

Comment:
Tauricia was sacrosanct to the Hypsistarians and had little interest from the Hellenics. The Hypsistarian religion began in Tauricia, and Tauricia was largely Hypsistarian country. There shouldn't be a Hellenic interest in this title, TBH.